docs/FURPS/communities-over-waku.md

```diff
@@ -0,0 +1,51 @@
+# Waku Requirements for Status Communities — FURPS
+
+Note: These requirements assume the Community Description has been moved to Codex
+
+## Functionality
+
+- **Community messages** (channel posts, updates) must be sent and received via **Waku**.
+- **Description change notifications** must be broadcast on Waku when the community description is updated via Codex.
```

Comment:
```suggestion
- Participants must be notified of **Description changes**.
```

"Broadcast on Waku" implies that that all nodes need to be notfied rather than limited to  community members.